### DISCUSSION TITLE
lesson 7 FundMeTest.t.sol, regarding testMinimumDollarIsFive() function

### DISCUSSION THREAD

**Discussion by grmihadogr** [Cyfrin/foundry-full-course-cu#398] [Q&A] [answered]
Hi team,
I have a question regarding  function testMinimumDollarIsFive() as follow.  

```solidity
 function testMinimumDollarIsFive() public {
        assertEq(fundMe.MINIMUM_USD(), 5 * 1e18); // why add () fix the problem
    }
```
Q1. Why should I add "()" after fundMe.MINIMUM_USD? fundMe.MINIMUM_USD was defined as a constant instead of a function in FundMe.sol.
```solidity
uint256 public constant MINIMUM_USD = 5 * 1e18;
```
Q2. I tried to write the test function as follow. It is not a assert. Does this tell me anything?
```solidity
  function testMinimumDollarIsFive2() public view returns (uint256) {
        return (fundMe.MINIMUM_USD());
    }
```
Any comment is appreciated.

**Comment by Naman1729** [comment, marked as answer]
Hey @grmihadogr ,
Ans 1 : ) when we declare a state variable as public, the compiler automatically generates a getter function for that variable. This getter function allows us to access the value of the variable from outside the contract.

Ans 2 : ) In my opinion, this function pass. but , it doesn't test anything only see that function is return uint256 value . so, this doesn't have any Significance in FundMeTest.t.sol.

**Reply by grmihadogr** [reply]
Thanks! It is insightful!

**Comment by DCVglobalnetwork** [comment]
so what function will actually work ?
